mockserver-netty/src/main/java/org/mockserver/netty/proxy/BinaryRequestProxyingHandler.java

```diff
@@ -123,7 +188,8 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
             mockServerLogger.logEvent(
                 new LogEntry()
                     .setLogLevel(Level.ERROR)
-                    .setMessageFormat("exception caught by " + this.getClass() + " handler -> closing pipeline " + ctx.channel())
+                    .setMessageFormat(
+                        "exception caught by " + this.getClass() + " handler -> closing pipeline " + ctx.channel())
```

Comment:
Can you expand your line length setting so that these sorts of files don't get formatted onto one line.  Its not a biggy but its more consistent with the rest of the code if this type of line is not broken.

Reply:
ooopsie, will do

Reply:
hmm, my setting is already at 120. what should i use?